ts-lib/package.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "neareth",
+  "version": "1.0.0",
+  "description": "The smart contract exposes two methods to enable storing and retrieving an encrypted (key, value)",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "lint": "eslint && prettier --check .",
+    "lint:fix": "eslint && prettier --write ."
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@nearfoundation/near-js-encryption-box": "^0.1.2",
+    "bs58": "^5.0.0",
+    "ethers": "^6.10.0",
+    "near-api-js": "^3.0.2",
+    "near-seed-phrase": "^0.2.0"
```

This is unused and can be removed
```suggestion
    "near-api-js": "^3.0.2"
```